addons/default/visiosoft/advs-module/resources/views/list/gallery.twig

```diff
@@ -24,12 +24,17 @@
                         </a>
                         {% if not setting_value('visiosoft.module.advs::hide_ad_cat') %}
                             <a href="{{ adv.detail_url }}" class="product-name-hb text-truncate text-decoration-none text-dark">{{ adv.cat1_name }} /{{ adv.cat2_name }}</a>
+
                         {% endif %}
                         {% if showPrice %}
-                            <div class="product-price-hb rounded d-flex align-items-center px-lg-3 px-2 mt-2 text-dark justify-content-between">
+                            <div class="product-price-hb rounded d-flex align-items-center mt-2 bg-secondary-md bg-none text-dark justify-content-between">
                                 {{ adv.price != '0' ? currency_format(adv.price,adv.currency) : trans('visiosoft.module.advs::field.free') }}
                                 {{ addBlock('list/extra-actions', {'ad': adv, 'vars': _context}, ['comparisons'])|raw }}
                             </div>
+                            <div class="review">
+                                <button class="m-0 btn btn-sm btn-outline-dark d-md-none d-block w-100">{{ trans('visiosoft.module.advs::field.add_to_cart') }}</button>
+                            </div>
```

Comment:
Use addBlock

Reply:
@fatihalp how am I supposed to use "addBlock"?

Reply:
explained

Reply:
actually at that stage we dont need add to card here and not ready for it @metehancelik 

you can remove it

Reply:
alright, will do it.